.storybook/seed-fake.js

```diff
@@ -56,6 +56,32 @@ export const seedFake = (db) => {
       image_alt_text: 'August Wifi Smart Lock 3rd Gen, Silver, Front',
       code_constraints: [],
     },
+    errors: [
```

Comment:
I planned to remove this before merging, but maybe we want to add some constructs for simulating device errors in a cleaner way? Updating the data here requires a restart of the Storybook server

Reply:
> I will remove this before merging,

Not sure you need to. Maybe leave 1, or 2? I believe adding data to the fake (server data) is the preferred approach on this project.